cmd/zcertificate/main.go

```diff
@@ -68,12 +72,14 @@ func processCertificate(in <-chan []byte, out chan<- []byte, wg *sync.WaitGroup)
 			if crashIfParseFail {
 				log.Fatal("parsing errors are fatal")
 			}
-			continue
-		}
+			if !jsonIfParseFail {
+				continue
+			}
+                }
+                // The certificate was parsed (or not). Run ZLint on it.
+                zlintResult := zlint.LintCertificate(parsed)
```

Comment:
I think this happens to work today because ZLint returns `nil` [when given `c == nil`](https://github.com/zmap/zlint/blob/f46d09c4ec0f3e1036731915a4d61938004b2d76/v2/zlint.go#L51-L53) but I think it would be better overall to express this like:

```go
var zlintResult *zlint.ResultSet
if parsed != nil {
  zlintResult = zlint.LintCertificate(parsed)
}
```